chaincode/.env.example

```diff
@@ -0,0 +1,32 @@
+ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
```

Comment:
There is a lot of stuff here. I hope we can simplify to just a few later that's absolutely needed for production. Move test-related variables to a separate example file?

Reply:
Removed the unused variables and created a test object.
Could also remove the non Harmony URL's.
For now I've left them in, in case we want to deploy to other networks.